ui/src/app/edge/live/common/grid/modal/modal.html

```diff
@@ -1,29 +1 @@
-<oe-modal *ngIf="isInitialized" [title]="'General.grid'| translate">
-
-    <oe-modal-line *ngIf="grid.mode === GridMode.OFF_GRID" [name]="'General.offGrid' | translate">
-    </oe-modal-line>
-
-    <oe-modal-line [name]="'General.gridBuyAdvanced' | translate" [value]="grid.buyFromGrid"
-        [converter]="Utils.CONVERT_TO_WATT">
-    </oe-modal-line>
-    <oe-modal-line [name]="'General.gridSellAdvanced' | translate" [value]="grid.sellToGrid"
-        [converter]="Utils.CONVERT_TO_WATT">
-    </oe-modal-line>
-
-    <ng-container *ngFor="let meter of meters">
-        <oe-modal-line *ngIf="meters.length > 1" [name]="meter.component.alias ?? meter.component.id"
-            [channelAddress]="meter.component.id + '/ActivePower'" [converter]="Utils.CONVERT_TO_WATT">
-        </oe-modal-line>
-
-        <!-- If meter is asymmetric, show Phases -->
-        <ng-container *ngIf="meter.isAsymmetric">
-            <oe-modal-meter-phases [component]="meter.component" [setTranslatedName]="setTranslatedName">
-            </oe-modal-meter-phases>
-        </ng-container>
-    </ng-container>
-
-    <!-- Info Text -->
-    <oe-modal-info-line [info]="'Edge.Index.Widgets.phasesInfo' | translate">
-    </oe-modal-info-line>
-
-</oe-modal>
\ No newline at end of file
+<formly-form style="display: block; height: inherit" [form]="form" [fields]="fields"></formly-form>
```

Comment:
This should also be wrapped to a separate component - or we even create a global HTML file with this content that can be used directly as `templateUrl`